bitnami/keycloak/templates/job.yaml

```diff
@@ -0,0 +1,81 @@
+---
+{{- if .Values.KcCasC.enabled }}
+apiVersion: batch/v1
```

Comment:
Could you try adding the same type of standard values that we can see in statefulset.yaml?